src/components/tree/tree.js

```diff
@@ -41,6 +41,12 @@ class Tree extends React.Component {
         root: [0, 0]
       }));
     };
+    /* pressing the escape key should dismiss an info modal (if one exists) */
+    this.handlekeydownEvent = (event) => {
+      if (event.keyCode===27 && this.state.selectedNode) { // key 27 is esc
```

Comment:
FYI, `keyCode` has been [marked as deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). 

The alternative is to use `key`, which is equal to "Escape" or "Esc" in older browsers (see [key values](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values)):